### ISSUE TITLE
[BUG] method fill_missing_values changes columns name

### ISSUE THREAD

**Issue by PengfeiWangWZ** [unit8co/darts#422]
**Describe the bug**
Method `fill_missing_values` changes column name in version 0.9.1, while it doesn't in older version like 0.6.1.

**To Reproduce**
```
import pandas as pd
from darts.timeseries import TimeSeries
from darts.utils.missing_values import fill_missing_values

df = pd.DataFrame([
    {'date': '2021-01-01', 'value': 3},
    {'date': '2021-01-02', 'value': 2},
    {'date': '2021-01-03', 'value': None},
])

series = TimeSeries.from_dataframe(df, 
                                   time_col='date', 
                                   value_cols='value', 
                                   freq='D')
series = fill_missing_values(series=series, fill=0.0)

series.columns
# In version 0.9.1
Index(['0'], dtype='object', name='component')
```

**Expected behavior**
It should not change the column name.

**System (please complete the following information):**
 - Python version: 3.7
 - darts version: 0.9.1

**Comment by pennfranc** [comment]
Hi @PengfeiWangWZ, you're absolutely right, this bug got introduced fairly recently. We have already fixed it on the current version of the develop branch, and it will be ironed out in the official release soon!

**Comment by PengfeiWangWZ** [comment]
@pennfranc Thanks for looking into this and I'm looking forward to the newer version!

Btw, I will leave it to you to close the issue right now or wait until the next release. Thanks again.

**Comment by hrzn** [comment]
This should now be solved in 0.10.0
